File: beecrowd/go/bee1115.go
package main

import "fmt"

func main() {
	var x, y int

	fmt.Scanf("%d", &x)
	fmt.Scanf("%d", &y)

	for {
		if x > 0 && y > 0 {
			fmt.Printf("primeiro\n")
		}
		if x < 0 && y > 0 {
			fmt.Printf("segundo\n")
		}
		if x < 0 && y < 0 {
			fmt.Printf("terceiro\n")
		}
		if x > 0 && y < 0 {
			fmt.Printf("quarto\n")
		}

		fmt.Scanf("%d", &x)
		fmt.Scanf("%d", &y)

		if x == 0 || y == 0 {
			break
		}

	}

}

File: beecrowd/c/bee1115.c
#include <stdio.h>

int main() {

	int x, y;

	scanf("%d", &x);
	scanf("%d", &y);

	for (;;) {
		if (x > 0 && y > 0) {
			printf("primeiro\n");
		}
		if (x < 0 && y > 0) {
			printf("segundo\n");
		}
		if (x < 0 && y < 0) {
			printf("terceiro\n");
		}
		if (x > 0 && y < 0) {
			printf("quarto\n");
		}
  
		scanf("%d %d", &x, &y);
  
		if (x == 0 || y == 0) return 0;
	}

}
